modules/augment/include/opencv2/augment/rotate.hpp

```diff
@@ -0,0 +1,55 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+
+#ifndef OPENCV_AUGMENT_ROTATE_HPP
+#define OPENCV_AUGMENT_ROTATE_HPP
+#include <opencv2/augment/Transform.hpp>
+
+namespace cv {
+namespace augment {
+
+    //! @addtogroup augment
+        //! @{
+
+    class CV_EXPORTS_W Rotate : public Transform
+    {
+    public:
+        /* @brief Constructor
+           @param _angleRange the range of angle values to rotate the image
+        */
+        CV_WRAP Rotate(Range _angleRange);
+
+        /* @brief Constructor to initialize the rotation transformation with default angleRange
+        */
+        CV_WRAP Rotate();
```

Comment:
What means "default angleRange"? We need to have two options: rotate to specific angle or rotate randomly by the angle selected from range. So I think second construction should be like `Rotate(float angle = 0)`.

Reply:
I think fixed angle is easy to specify right now.. the user would just use a range that starts and ends with the same value.. we wouldn't need a special constructor for that.. don't you think?

Reply:
I agree that we can implement it in this way internally. However for users we need to provide as minimal API as possible.

Reply:
Yes, that's why I think the user should only have to deal with class Augmenter, to provide an api as high as possible

Reply:
So there is still no constructor with a single specific angle.

Reply:
done